Project.toml

```diff
@@ -19,10 +19,10 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Wavelets = "29a6e085-ba6d-5f35-a997-948ac2efa89a"
 
 [compat]
-AbstractFFTs = "= 0.4.1, 0.5, 1.0"
+AbstractFFTs = "1"
 DSP = "^0.5.2, 0.6, ^1, 0.7"
 DelayEmbeddings = "2"
-Distances = "0.8, 1"
+Distances = "0.10, 1"
```

Comment:
Might as well drop this to just "1".

Reply:
I just saw that v0.10.7 is the latest version for Distances.jl. I guess I put 1.0 here to be ready for the 1.0 release at some point. Maybe just keep it as is, so no changes are needed when 1.0 is released? Or delete 1.0 from the requirements, since there might be breaking changes when Distances 1.0 is released?

Reply:
oh yeah, I haven't realized that.